embulk-core/src/main/java/org/embulk/spi/json/JsonParser.java

```diff
@@ -45,14 +55,19 @@ private static String sampleJsonString(String json) {
     }
 
     private static class StreamParseContext extends AbstractParseContext implements Stream {
-        public StreamParseContext(JsonFactory factory, InputStream in) throws IOException, JsonParseException {
-            super(createParser(factory, in));
+        public StreamParseContext(JsonFactory factory, InputStream in, String pointer) throws IOException {
+            super(createParser(factory, in, pointer));
         }
 
-        private static com.fasterxml.jackson.core.JsonParser createParser(JsonFactory factory, InputStream in)
-                throws IOException {
+        private static com.fasterxml.jackson.core.JsonParser createParser(
+                JsonFactory factory, InputStream in, String pointer
+        ) throws IOException {
```

Comment:
nitpick: Ah, I've never seen this style, but our checkstyle accepted this... ;)

I usually have done it like:
```
        private static com.fasterxml.jackson.core.JsonParser createParser(
                JsonFactory factory, InputStream in, String pointer) throws IOException {
```
or,

```
        private static com.fasterxml.jackson.core.JsonParser createParser(
                JsonFactory factory,
                InputStream in,
                String pointer)
                throws IOException {
```

Reply:
OK, I fixed the style to the first one.